app/services/facet.service.js

```diff
@@ -129,22 +132,22 @@
       for (var i in this.facets[key]) {
         var result;
         var filter = this.facets[key][i].value;
-        // filter is a function
         if (filter.call) {
+          // filter is a function
```

Comment:
Moved back. I find it a bit confusing to have a comment at the top of an if-else ladder that only applies to the first segment of it, but it's a mild preference.